backend/Recodo.BLL/Services/UserService.cs

```diff
@@ -31,5 +41,89 @@ public async Task<UserDTO> CreateUser(NewUserDTO userRegisterDTO)
 
             return _mapper.Map<UserDTO>(userEntity);
         }
+
+        public async Task UpdateUser(UpdateUserDTO userDto, IFormFile avatar)
+        {
+            var userEntity = await _context.Users.FirstOrDefaultAsync(u => u.Id == userDto.Id);
+            if (userEntity == null) { return; }
+
+            userEntity.WorkspaceName = userDto.WorkspaceName ?? userDto.WorkspaceName;
+
```

Comment:
is it correct?
`userEntity.WorkspaceName = userDto.WorkspaceName ?? userDto.WorkspaceName;`